sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/DefaultChannelTest.java

```diff
@@ -1071,4 +1086,164 @@ public void scheduleLogsWithNewToken() {
         listenerArgumentCaptor.getValue().onNewAuthToken("test");
         verify(channel, times(2)).checkPendingLogs(any(DefaultChannel.GroupState.class));
     }
+
+    @Test
+    public void checkPendingLogsCheckDelay() throws InterruptedException {
```

Comment:
Those tests are never checking what batches we are actually sending. We need a unit test that  does steps similar to `analyticsSuccess` but with non default interval and check that we don't send batches too early when we have a lot of logs. (we need the 200 logs test basically with custom interval).